src/api.js

```diff
@@ -659,13 +668,6 @@
       });
   };
 
-  Contact.prototype.destroy = function (callbacks) {
```

Comment:
I don't think we should delete this method entirely. We should add a warn log that it is deprecated and will be removed in the next major version release.

Reply:
Although, I suppose it hasn't been working for a while because its not in the client file anymore. @h3adache what are your thoughts on this? This api will fail if you call it right now. Should we just remove it?

Reply:
Since the call to it was already failing, removing it and added a note to the release notes should be fine.
This is a grey area for versioning as typically you should just noop it and document that it wasn't expected to work + some work around.
Curious though we have tickets around destroy that seem like it should work?

https://github.com/amazon-connect/amazon-connect-streams/issues/320

Reply:
Connection.prototype.destroy works. Contact.prototype.destroy does not

Reply:
-1 on my reading stats. 

+1 Let's noop this, document it's deprecation and add a warning on use.

Reply:
Added it back with deprecation mark and warning message